specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -91,42 +91,51 @@ export function StatsPage(): JSX.Element | null {
     state.type === 'EditingState' ||
     isAddingItem ||
     state.type === 'PageRenameState';
-  const isEditingCollection = React.useRef(false);
+
+  const hasEditPermission = React.useMemo(
```

Comment:
Don't overuse useMemo when there is no need to - see https://blog.logrocket.com/rethinking-hooks-memoization/